### DISCUSSION TITLE
Get current icon name?

### DISCUSSION THREAD

**Discussion by fractaltheory** [skb1129/react-native-change-icon#34] [Ideas]
Hi, thanks for the great package. Was wondering if there were any plans to include a method to get the current icon name? This would be helpful to indicate to the user which icon is currently selected, and to prevent them from selecting it again. Similar packages implement this for iOS - but unsure how it would be done for Android.

**Comment by skb1129** [comment]
@fractaltheory There are no plans to make this change.
While using this package in React-Native, we usually store the current icon information in the AsyncStorage for easier and faster access because we already have to store the names of all possible icons that can be switched in order to call the native function.
Let me know if you can solve your issue this way otherwise we can consider adding such functionality to the package.

**Reply by fractaltheory** [reply]
Thanks for the reply and suggestion. That definitely works so no need for the feature, though I do think it would be handy :)

**Comment by skb1129** [comment]
@fractaltheory getIcon is added in the package, thanks to @TheKeeroll
